R/semver-handling.R

```diff
@@ -0,0 +1,93 @@
+#' Get max version
+#' 
+#' Extract max version from vector of version strings.
+#' 
+#' @param version_strings vector of string containing version string.
+#' 
+#' @return string representing the max version found
+get_max_version <- function(versions_strings) {
+  versions_list <- list()
```

Comment:
Why are you using a list as opposed to a vector?

Reply:
A vector can only store one dimensional same type objects. This stores list objects returned by my semver function.